src/Infrastructure/BotSharp.Abstraction/Browsing/Models/PageActionArgs.cs

```diff
@@ -47,4 +47,7 @@ public class PageActionArgs
     public int WaitTime { get; set; }
 
     public bool ReadInnerHTMLAsBody { get; set; } = false;
+    public bool IsScrollPage { get; set; } = false;
```

Comment:
Why we don't use `ScrollPage`?
![image](https://github.com/user-attachments/assets/e2731621-c939-48da-a4e7-ee6531d233b6)